lib/sequence.js

```diff
@@ -53,6 +53,18 @@ export default class Sequence {
       });
   }
 
+  static async getLatestMigration(fs, dirPath) {
+    const sequence = await this.readMigrations(fs, dirPath);
+
+    for (let i = sequence.programs.length - 1; i >= 0; i--) {
```

Comment:
Thus we start at the back and return the first time we find an ordinal (aka version). It's possible for it to not have a version (bluefin allows for a prologue and an epilogue) so ignore those.